scripts/helpers.py

```diff
@@ -89,7 +89,30 @@ def get_existing_prs(config: configparser.ConfigParser):
     return prs
 
 
-def commit_message(path, old, new, rel_path, version="master"):
+def get_o_spreadsheet_release_prs(config: configparser.ConfigParser):
+    prs = {}
+    path = config["spreadsheet"]["repo_path"]
+    with pushd(path):
+        for version in spreadsheet_odoo_versions.keys():
+            existing_prs = subprocess.check_output(
+                [
+                    "gh",
+                    "pr",
+                    "list",
+                    "--search",
+                    f"base:{version} is:open [REL]in:title",
+                ]
+            ).decode("utf-8")
+            if existing_prs:
+                url = "https://github.com/odoo/%s/pull/%s" % (
+                    "o-spreadsheet",
```

Comment:
There's no need to insterpolate "o-spreadsheet"
`"https://github.com/odoo/o-spreadsheet/pull/%s`